src/neuroglancer/synapse/frontend.ts

```diff
@@ -0,0 +1,309 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
```

Comment:
From Jeremy: They also should include the standard license header. If you don't want it to say Copyright Google Inc., as that is technically inaccurate, it can instead say "The Neuroglancer Authors". My understanding (from our legal dept.) is that legally this makes no difference, as you retain the copyright in either case.